packages/babel-plugin/src/styled/__tests__/tagged-template-expression.test.tsx

```diff
@@ -656,7 +656,7 @@ describe('styled tagged template expression', () => {
     ]);
   });
 
-  it('should only destructure a prop if hasnt been already', () => {
+  it.skip('should only destructure a prop if hasnt been already', () => {
```

Comment:
TODO: The example provide has now a different output and is not longer relevant. May need to replace it